src/test/kotlin/com/makarytskyi/rentcar/repository/OrderRepositoryTests.kt

```diff
@@ -0,0 +1,191 @@
+package com.makarytskyi.rentcar.repository
+
+import fixtures.CarFixture.randomCar
+import fixtures.OrderFixture.monthAfter
+import fixtures.OrderFixture.monthAndDayAfter
+import fixtures.OrderFixture.randomOrder
+import fixtures.OrderFixture.unexistingOrder
+import fixtures.UserFixture.randomUser
+import java.util.Date
+import kotlin.test.assertNotNull
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+
+internal class OrderRepositoryTests : ContainerBase {
+
+    @Autowired
+    lateinit var orderRepository: OrderRepository
+
+    @Autowired
+    lateinit var carRepository: CarRepository
+
+    @Autowired
+    lateinit var userRepository: UserRepository
+
+    @Test
+    fun `create should insert order and return it with id`() {
+        // GIVEN
+        val car = carRepository.create(randomCar())
+        val user = userRepository.create(randomUser())
+
+        // WHEN
+        val order = orderRepository.create(unexistingOrder(car.id, user.id))
+
+        // THEN
+        val foundOrder = orderRepository.findById(order.id.toString())
+        assertNotNull(foundOrder)
+        assertNotNull(order.id)
+    }
+
+    @Test
+    fun `findAll should find all orders`() {
+        // GIVEN
+        val car1 = carRepository.create(randomCar())
+        val user1 = userRepository.create(randomUser())
+        val order1 = orderRepository.create(randomOrder(car1.id, user1.id))
+
+        val car2 = carRepository.create(randomCar())
+        val user2 = userRepository.create(randomUser())
+        val order2 = orderRepository.create(randomOrder(car2.id, user2.id))
+
+        // WHEN
+        val orders = orderRepository.findAll()
+
+        // THEN
+        assertTrue(orders.any { it.id == order1.id && it.car?.id == order1.carId && it.user?.id == order1.userId })
```

Comment:
Provide a message and in case one of the conditions is false. Which one? How do you know?